### PR TITLE
fixed cleanup order to uninstall operator before cleaning sa to fix intermittent failures

### PR DESCRIPTION
**Pull request by marinakog**
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2149/

### CODE REVIEW

**Review by rjeberhard — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java [hunk 1]
```diff
@@ -482,10 +482,10 @@ public void testCreateSecondOperatorUsingSameOperatorNsNegativeInstall() {
       assertNull(opHelmParam2,
           "FAILURE: Helm installs operator in the same namespace as first operator installed ");
     } finally {
-      deleteSecret("ocir-secret",opNamespace);
-      cleanUpSA(opNamespace);
       uninstallOperator(opHelmParams);
       uninstallOperator(op2HelmParams);
+      deleteSecret("ocir-secret",opNamespace);
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java:487]: use REPO_SECRET_NAME in TestConstants for ocir-secret through the file

**Review by vanajamukkara — CHANGES_REQUESTED** [review]
Please update the PR with Jenkins results when you have them.

**Comment by anpanigr** [comment]
Run the entire suites on Jenkin and post the result.

**Review by anpanigr — APPROVED** [review]
LGTM

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java]: done

**Review by vanajamukkara — APPROVED** [review]

**Pull request merged**